### ISSUE TITLE
Layer Events Not Declared in LinkEventName Enum

### ISSUE THREAD

**Issue by ggeigs** [plaid/react-native-plaid-link-sdk#693]
# The problem

`LAYER_READY` and `LAYER_NOT_AVAILABLE` are missing from the LinkEventName enum. These event names are necessary for handling specific link events but are not available in the current enum declaration.

# Environment

|                                                  |                                                                                  |
| -----------------------------| ----------------------------------------------- |
| Plaid Link React Native          | 11.11.1                                                                 |
| ReactNative Version               | 0.73.4                                                                 |
| Occurs on Android                  | yes                                                                     |
| Android OS Version                 | 14.0.0 (34)                                                         |
| Android Devices/Emulators    | Pixel 7a physical                                                |
| Occurs on iOS                          | yes                                                                     |
| iOS Version                               | 17.4                                                                    |
| iOS Devices/Emulators            | iPhone 15                                                        |

# Steps to Reproduce

- Install version 11.11.1 of react-native-plaid-link-sdk.
- Try to handle `LAYER_READY` and `LAYER_NOT_AVAILABLE` via `usePlaidEmitter`.
- Observe that these event names are not declared in the enum, leading to errors.

# Expected Result
The LinkEventName enum should include LAYER_READY and LAYER_NOT_AVAILABLE to handle these events properly.

# Screenshots
![image](https://github.com/user-attachments/assets/c90d6001-bf8d-4a5c-81cc-e11e8b3b9730)

# Logs
N/A

# Code To Reproduce Issue 
```
  usePlaidEmitter((event: LinkEvent) => {
    switch (event.eventName) {
      case LinkEventName.LAYER_READY:
        // Open Link
        break;
      case LinkEventName.LAYER_NOT_AVAILABLE:
        // Run another fallback flow
        break;
      default:
        // Other cases ignored in this use case
        break;
    }
  });
```

**Comment by dtroupe-plaid** [comment]
@ggeigs Thanks for reporting.  I will get this resolved ASAP

**Comment by dtroupe-plaid** [comment]
@ggeigs this should be resolved in `11.11.2 ` I'm going to close this issue, but please let me know if you're still experiencing issues and I'll reopen.
